### ISSUE TITLE
Trouble with  wala.properties settings.

### ISSUE THREAD

**Issue by caoyan66** [wala/WALA#138]
Hi,

Sorry for not understanding the wala.properties settings. 

I have been trying WALA in both maven and ``non-maven'' way.

1. Using Wala via Maven
I got the error message as below,
![image](https://cloud.githubusercontent.com/assets/6722880/23367505/7830fe4a-fcd8-11e6-8dd5-5b47732f54ec.png)

I had read the user guide, in which it states that 

```
Configuring WALA properties
You will need to set up a few Java properties files before you can run the WALA code.
In the com.ibm.wala.core project, you need to copy the file dat/wala.properties.sample to dat/wala.properties. You need to then edit wala.properties to reflect your environment. See the properties file for the detailed instructions on what properties you must set. For beginners, we recommend you set the java_runtime_dir property (which is mandatory) and the output_dir property (required for some tests below like PDFTypeHierarchy). Note that the directory specified for output_dir must exist on the filesystem; WALA will not create it. In the com.ibm.wala.core.tests project, you need to copy the file dat/wala.examples.properties.sample to dat/wala.examples.properties. For the moment, there are no mandatory settings in this file that you need to worry about, so move on. We will describe a few of the other optional properties shortly.
```

Sorry for my stupidity, I am not sure how to do so. I tried to add the wala.properties under my project/resource folder, but not work.

2. In `non-waven` way, after I had included all the jar files I needed, I got the error message as below.

java.io.IOException: property_file_unreadable wala.properties
	at com.ibm.wala.properties.WalaProperties.loadPropertiesFromFile(WalaProperties.java:121)
	at com.ibm.wala.properties.WalaProperties.loadProperties(WalaProperties.java:92)
	at com.ibm.wala.properties.WalaProperties.getJ2SEJarFiles(WalaProperties.java:46)

I googled for the problem as well as read the troubleshooting user guide but not be able to figure out how. :-(. IN user guide, it states that:

 " The Eclipse build process is supposed to copy the contents of com.ibm.wala.core/dat to com.ibm.wala.core/bin when the com.ibm.wala.core project is built. More than one user has reported that this copy does not happen their Eclipse environment.  ... "

![image](https://cloud.githubusercontent.com/assets/6722880/23367330/e1ada298-fcd7-11e6-8591-30714b83cb4d.png)

but I cannot even locate the com.ibm.wala.core/dat or com.ibm.wala.core/bin in y jar files.

SO please, would you mind helping me with this issue, please? Hope can hear from you soon.

Thanks!

**Comment by msridhar** [comment]
@caoyan66 sorry about this.  The Maven issue is a bug for sure.  We will try to cut a new release ASAP.  In the meantime, if you pull in the 1.3.10-SNAPSHOT version of WALA instead of 1.3.9 from Maven, I think at least the Maven way should work.  Can you try that?

**Comment by caoyan66** [comment]
Hi,

Yes, I was able to pull in the 1.3.10-SNAPSHOT version of WALA, and my issues is resolved. thanks!

**Comment by msridhar** [comment]
This is fixed if you use the 1.4.0 release, which is now on Maven Central.
